package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "todo-list-app",
+  "version": "1.0.0",
+  "description": "A todo-list app for adding and removing items",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack --config webpack.config.js"
+  },
```

Comment:
Great job @kennyegun24 🤝

- [ ] Kindly make sure your server is well installed because when I start the server. The process quits without rendering the intended page.
![server](https://user-images.githubusercontent.com/63917725/192758090-1ef47136-ce84-4537-b3b5-8a588d75b2a0.PNG)